test/spix/smoke_test.py

```diff
@@ -103,6 +103,10 @@ def test_start_app(app, screenshot_path, extra, process_alive):
     extra.append(extras.html('<img src="images/startup.png"/>'))
 
 
+@pytest.mark.skipif(
+    platform.system() != "Linux",
+    reason="PostGIS test requires a docker linux container",
```

Comment:
For the record: macos would likely work as well (kartoza/postgis even publishes arm64 images unlike postgis/postgis)